.ci/scripts/docker-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -xueo pipefail
+
+export
+id
+node --version
+npm --version
+npm install
+npm list
+
+# Let's store the test output in the target folder
+TARGET_FOLDER=target
+mkdir -p target
+nyc npm test | tee ${TARGET_FOLDER}/test-suite-output.tap || true
```

Comment:
Let's fail the build as we do with the nodejs agent:
- https://github.com/elastic/apm-agent-nodejs/blob/master/.ci/scripts/docker-test.sh#L17
```suggestion
nyc npm test | tee ${TARGET_FOLDER}/test-suite-output.tap
```